plugin/trino-bigquery/src/test/java/io/trino/plugin/bigquery/TestBigQueryInstanceCleaner.java

```diff
@@ -60,6 +60,35 @@ public void setUp()
         this.bigQuerySqlExecutor = new BigQuerySqlExecutor();
     }
 
+    @Test
+    public void cleanUpDatasets()
+    {
+        LOG.info("Identifying datasets to drop...");
+        // Drop all datasets with our label created before yesterday (in UTC)
```

Comment:
Maybe we can just "created more than 24 hours ago". That would avoid mentioning time zones